app.py

```diff
@@ -435,21 +517,35 @@ async def get_repositories(project: dict[str, Any]):
                 for repo in repositories_batch
             ]
         )
-
+        (
+            [
+                await get_or_create_bitbucket_webhook(
+                    project_key=project["key"],
+                    repo_key=repo["slug"],
+                    webhook_url=port_webhook_url,
+                    events=WEBHOOK_EVENTS,
+                )
+                for repo in repositories_batch
+            ]
+            if IS_VERSION_8_7_OR_OLDER
+            else None
+        )
         await get_repository_pull_requests(repository_batch=repositories_batch)
 
+
 async def get_repository_pull_requests(repository_batch: list[dict[str, Any]]):
     pr_params = {"state": "ALL"}  ## Fetch all pull requests
     for repository in repository_batch:
-        pull_requests_path = f"projects/{repository['project']['key']}/repos/{repository['slug']}/pull-requests"
+        pull_requests_path = f"projects/{repository['project']['key']}/repos/{repository['slug']}/pull-requests?state=all"
```

Comment:
ingesting all pull requests using `state=all` will make the app very slow since it will attempt to bring very old and merged PRs. Let's keep the default by removing this state=all

Reply:
If this is removed, only the open PRs will be ingested and the `closedDate` will not be in the payload

Reply:
I guess I will have to modify the webhook config to listen for this then